scripts/coderabbit_retry/tests/test_coderabbit_retry.py

```
@@ -0,0 +1,361 @@
+        assert is_wip_title(title=title) is False
+
+
+class TestProcessPr:
+    """Tests for process_pr orchestration."""
+
+    @patch("scripts.coderabbit_retry.coderabbit_retry.trigger_review", return_value=True)
+    @patch("scripts.coderabbit_retry.coderabbit_retry.has_review_request_after", return_value=False)
+    @patch("scripts.coderabbit_retry.coderabbit_retry.time_time")
+    @patch("scripts.coderabbit_retry.coderabbit_retry.check_rate_limit")
+    def test_triggers_when_wait_elapsed(
+        self, mock_check: MagicMock, mock_time: MagicMock, mock_review: MagicMock, mock_trigger: MagicMock
```

Ruff (0.16.1)

[warning] Unused method argument: `mock_review`

(ARG002)